microsoft-edge/toc.yml

```diff
@@ -496,11 +496,11 @@
 # Performance tool ------------------------------------------------------------
             - name: Performance tool
               items:
-                - name: Get started analyzing runtime performance
+                - name: Introduction to the Performance tool
                   href: devtools-guide-chromium/evaluate-performance/index.md
                   displayName: Performance tool
 
-                - name: Analyze runtime performance
+                - name: Troubleshooting common performance issues
                   href: devtools-guide-chromium/rendering-tools/index.md
                   displayName: Performance tool
```

Comment:
note that if you want more lookup index entries, if "runtime performance" is a term people will enter in the multi-purpose TOC text box, can use a comma-delim'd list: 
```
displayName: Performance tool, runtime performance
```
I question whether the term "runtime performance" is helpful, or adds as much confusion as clarity.